include/qb/qbloop.h

```diff
@@ -36,6 +36,16 @@ extern "C" {
  *
  * Main loop manages timers, jobs and polling sockets.
  *
+ * To be noted that local implementation of the event loop does care about
+ * priorities, these are rather advisory, however.  For the true real-world
+ * systems with requirements of deterministic responses, you may be better
+ * served with these priorities strictly separated and abided, which is the
+ * case, e.g., with GLib.  On the other hand, for early stages and timing
```

Comment:
I don't really have too high opinion about libqb loop, but it may be helpful to be a little more specific. What do you mean by "... are rather advisory"? Why it doesn't fulfill "deterministic" guarantees (and maybe a small note why Glib does so?)? Also are you sure that your advice is really true for all (or at least most of the) "real-world" systems?

Reply:
The wording there is hard to understand for me, it's not in idiomatic modern English. I'll work out a better phrasing and let you know what I come up with.